backend/src/zimfarm_backend/db/models.py

```diff
@@ -190,6 +189,48 @@ class Task(Base):
     )
     offliner_definition: Mapped["OfflinerDefinition"] = relationship(init=False)
 
+    files: Mapped[list["File"]] = relationship(
+        back_populates="task", cascade="all, delete-orphan", init=False
+    )
+
+
+class File(Base):
+    __tablename__ = "file"
+    id: Mapped[UUID] = mapped_column(
+        init=False, primary_key=True, server_default=text("uuid_generate_v4()")
+    )
+    name: Mapped[str]
+    status: Mapped[str]
+    size: Mapped[int | None] = mapped_column(default=None, type_=BigInteger)
+
+    # CMS status fields (nested in TaskFileSchema.cms)
+    cms_status_code: Mapped[int | None] = mapped_column(default=None)
```

Comment:
You should drop:
- `cms_status_code`: no need to keep failure status code in DB, it should not happen, will never be analyzed from UI, ops will go to the logs
- `cms_succeeded`: it is confusing and a duplication of `cms_notified` ; let's keep only `cms_notified` for the time being (and populate it to "False" for now in the migration, we will restart CMS with a blank DB)
- `cms_book_id` and `cms_title_ident`: these information will not be supplied by the CMS anymore (this is a new info I've never shared with you)

Reply:
Consequence is that downgrade will not be capable to repopulate the "cms_book_id", "cms_title_ident" and "cms_status_code", but this is only garbage ATM anyway so I'm fine with it.